### ISSUE TITLE
Doesn't appear to properly handle two calls in one format() operation

### ISSUE THREAD

**Issue by RoryLambert** [AndrewChap/num2tex#1]
`'{0:.2e} + {1:.2e}'.format(0.00000012423,0.00000000000009)`  
returns;
`'1.24e-07 + 9.00e-14'`

However, when using num2tex;
`'{0:.2e} + {1:.2e}'.format(num2tex.num2tex(0.00000012423),num2tex.num2tex(0.00000000000009))`
returns;
`'9.00 \\times 10^{-14} + 9.00 \\times 10^{-14}'`

**Comment by AndrewChap** [comment]
Thank you for reporting this!  I'll look into it how to implement the correct behavior.  My apologies for the delay, I had a daughter born in February and am just now getting back on GitHub.

**Comment by AndrewChap** [comment]
Fixed in release 0.8 on PyPI.  To get the update:

```bash
pip install --upgrade num2tex
```
Now works as expected:
```python
>>> import num2tex
>>> '{0:.2e} + {1:.2e}'.format(num2tex.num2tex(0.00000012423),num2tex.num2tex(0.00000000000009))
'1.24 \\times 10^{-7} + 9.00 \\times 10^{-14}'
```

Thank you for catching this!

**Comment by RoryLambert** [comment]
Congrats on the new addition! Thanks for this software, and thanks for the fix :)

On Jun 16 2019, at 1:18 am, Andrew Chap <notifications@github.com> wrote:
Fixed in release 0.8 on PyPI. To get the update:


pip install --upgrade num2tex

Now works as expected:


>>> import num2tex
>>> '{0:.2e} + {1:.2e}'.format(num2tex.num2tex(0.00000012423),num2tex.num2tex(0.00000000000009))
'1.24 \\times 10^{-7} + 9.00 \\times 10^{-14}'

Thank you for catching this!
